src/seedu/addressbook/AddressBook.java

```diff
@@ -217,6 +212,16 @@ public static void main(String[] args) {
      * signature anyway.
      * ====================================================================
      */
+	 
+	 public static void promptUserInput(){
+		 while (true) {
```

Comment:
Take note of the indentations here, the while(true) line is a tab space more than the rest of the code